axelor-sale/src/main/resources/views/SaleOrder.xml

```diff
@@ -299,7 +299,7 @@
 				<button name="generateProject" readonlyIf="!$_projectGeneratorType" title="Generate Project/Business" if="!__config__.app.getApp('business-project').automaticProject" showIf="project == null" hidden="true" onClick="save,action-business-project-method-generate-project" colSpan="12"/>
 			</panel>
 		</panel>
-	    <panel if="__config__.app.isApp('supplychain') &amp;&amp; __config__.app.getApp('supplychain').getAllowTimetableInvoicing()" if-module="axelor-supplychain" name="timetable" title="Timetable">
+	    <panel if="__config__.app.isApp('supplychain') &amp;&amp; __config__.app.getApp('supplychain').getAllowTimetableInvoicing() &amp;&amp; !__config__.app.getApp('supplychain').getManageInvoicedAmountByLine()" if-module="axelor-supplychain" name="timetable" title="Timetable">
```

Comment:
This should be done on SupplychainApp config if option  'Manage the invoiced amount by line' set  to true change 'Allow timetable invoicing' to false and make it readonly